py/tui/main.py

```diff
@@ -148,6 +150,12 @@ def run():
             try:
                 table_info, balance_info, fet_info, prot_info = serial.get_info()
                 cell_data = serial.get_cells()
+            except ValidationNotEqualError:
+                continue
+            except ValidationGreaterThanError:
+                continue
+            except ValidationLessThanError:
+                continue
```

Comment:
Instead of the specific errors it would be cleaner to catch the more general `ValidationFailedError`. 
Though the change should not even be necessary, see my other comment.